frontend/src/routes/index.svelte

```diff
@@ -141,53 +141,78 @@
       autofocus
     />
   </div>
-  <div class="sm:grid sm:grid-cols-2 sm:gap-2">
-    <MultiSelect
-      outerDivClass="bg-base-100"
-      --sms-options-bg={DaisyuiThemes[`[data-theme=${$chosenTheme}]`]["neutral-focus"]}
-      --sms-text-color={DaisyuiThemes[`[data-theme=${$chosenTheme}]`][
-        "neutral-content"
-      ]}
-      --sms-border="1pt solid {DaisyuiThemes[`[data-theme=${$chosenTheme}]`][
-        'primary'
-      ]}"
-      --sms-focus-border="2pt solid {DaisyuiThemes[`[data-theme=${$chosenTheme}]`][
-        'primary'
-      ]}"
-      --sms-active-color={DaisyuiThemes[`[data-theme=${$chosenTheme}]`]["primary"]}
-      --sms-remove-x-hover-focus-color={DaisyuiThemes[`[data-theme=${$chosenTheme}]`][
-        "base-content"
-      ]}
-      --sms-li-selected-bg={DaisyuiThemes[`[data-theme=${$chosenTheme}]`]["neutral"]}
-      --sms-li-active-bg={DaisyuiThemes[`[data-theme=${$chosenTheme}]`]["primary"]}
-      --sms-selected-color={DaisyuiThemes[`[data-theme=${$chosenTheme}]`]["primary"]}
-      bind:selected={selectedGenres}
-      on:change={debounceInput}
-      options={getFixedGenreValues(formattedGenres)}
+  <div
+    class="sm:grid sm:grid-cols-2 sm:gap-2 mt-2 mb-3"
```

Comment:
The selects do not align in mobile view.

Some value here is most likely the cause.

![image](https://user-images.githubusercontent.com/50198099/154952172-339b9785-cc49-4e8a-904c-aa1c3b9e7803.png)

Reply:
Furthermore when you change country, the providers are not visually reset. They still appear on the multiselect.

Reply:
Works now